src/library/scala/collection/StringParsers.scala

```diff
@@ -203,11 +203,11 @@ private[scala] object StringParsers {
         val len = endIndex - startIndex
         (len > 0) && {
           //the prefix part is
-          //hexDigits 
```

Comment:
I tried to remove these trailing whitespace changes but IntelliJ refuse to obey.

Reply:
That's OK. IntelliJ's rules are 1. don't harm humans 2. obey humans etc.